index.js

```diff
@@ -914,6 +915,52 @@ class KeyringController extends EventEmitter {
 
     return keyring;
   }
+
+  /*
+  Utility Methods
+  */
+
+  _uint8ArrayToString(mnemonic) {
+    const recoveredIndices = Array.from(
+      new Uint16Array(new Uint8Array(mnemonic).buffer),
+    );
+    return recoveredIndices.map((i) => wordlist[i]).join(' ');
+  }
+
+  _stringToUint8Array(mnemonic) {
+    const indices = mnemonic.split(' ').map((word) => wordlist.indexOf(word));
+    return new Uint8Array(new Uint16Array(indices).buffer);
+  }
+
+  _mnemonicToUint8Array(mnemonic) {
```

Comment:
Suppose it could be useful to go back and make this a public static method on `eth-hd-keyring` to avoid this duplication. cc @Gudahtt

Reply:
Good point. It would be ideal to eliminate this duplication.

If we're using this code in both places, maybe it would be better to move this to a third place? e.g. https://github.com/MetaMask/utils or `@metamask/scure-bip39`. Just a thought; `eth-hd-keyring` would suffice for now.

Reply:
yeah maybe `@metamask/scure-bip39` may make the most sense?

Reply:
Or we could drop this validation step. It seems to be redundant; `eth-hd-keyring` is doing it already here: https://github.com/MetaMask/eth-hd-keyring/blob/main/index.js#L284

Reply:
lol true. In which case `KeyringController` needn't depend directly on `@metamask/scure-bip39` at all!

Reply:
Oh! That would be good as well

Reply:
Removed `bip39` dependency altogether in latest commit: https://github.com/MetaMask/KeyringController/pull/179/commits/dea4b70574e4da2d72882801dbe7db0d1e93f1b9